rust/rocket/src/main.rs

```diff
@@ -22,6 +21,6 @@ fn main() {
         .port(3000)
         .finalize()
         .unwrap();
-    let app = rocket::custom(config, false);
+    let app = rocket::custom(config);
```

Comment:
This now enables logging. `.log_level(LoggingLevel::Off)` is the new way to disable logging.